src/error.rs

```diff
@@ -167,6 +167,22 @@ impl Error {
         }
     }
 
+    /// Ergonomic helper for handling all status codes as [Response].
+    ///
+    /// ```
+    /// # ureq::is_test(true);
+    /// let response = ureq::get("http://httpbin.org/status/500").call()
+    ///     .or_else(|e| e.into_response()).unwrap();
```

Comment:
Could you use a `?` instead of `unwrap`? That matches our other doctests better. It requires putting `# fn main() -> Result<(), ...>` at the top, and `# Ok(()` and `# }` at the bottom.